### ISSUE TITLE
Use CDN versions of common libraries

### ISSUE THREAD

**Issue by eddarmitage** [ocadotechnology/rapid-router#112]
I'm thinking of all the common-ish libraries we use (jQuery, Raphael, etc.). The advantages that I can see (in no particular order) are:
1. There's a chance users will already have the version we require cached
2. It takes a tiny bit of load off of our server
3. More requests can be done in parallel (See [here](http://webmasters.stackexchange.com/questions/26753/why-do-big-sites-host-their-images-css-on-external-domains))
4. The CDN can provide a copy of the resource that is physically closer to the user, although since this is a one-time thing it probably doesn't matter too much

Possibly look at [providing fallback](http://www.hanselman.com/blog/CDNsFailButYourScriptsDontHaveToFallbackFromCDNToLocalJQuery.aspx) to versions that the django app provides in its static directory for situations such as:
- CDN failures
- Local testing without an internet connection/cache.

I believe this could be done using something like [requirejs](http://requirejs.org/).

**Comment by j4mesholland** [comment]
No longer relevant
